packages/terra-site/src/examples/form/navigation.md

```diff
@@ -0,0 +1,23 @@
+## Components
+
+[ChoiceField](#choice-field)
```

Comment:
I decided to use the terra buttons, and do the scrollIntoView approach. From looking at https://github.com/ReactTraining/react-router/issues/394, it looks like that's the recommended approach.

https://github.com/cerner/terra-core/pull/305/commits/916aa0d7b5e2b244c2598284bb757011425af412